src/scripts/widgets/affix.coffee

```diff
@@ -10,3 +10,5 @@ $.fn.affix = () ->
     calculateAndSetNewPosition(startingTopPosition, ele)
     $(window).on 'scroll', () ->
       calculateAndSetNewPosition(startingTopPosition, ele)
+
+# TODO: Output a deprecation warning...
```

Comment:
Hmm... I did do this. It looks like I have lost some changes from Friday on my desktop. I need to reconcile the changes on both machines and push it back up. Give me a bit.